util/fipstools/test_fips.c

```diff
@@ -234,19 +234,19 @@ int main(int argc, char **argv) {
     goto err;
   }
   printf("About to AES-CTR Encrypt ");
-  hexdump(output, out_len);
+  hexdump(kPlaintext, sizeof(kPlaintext));
   AES_ctr128_encrypt(kPlaintext, output, sizeof(kPlaintext), &aes_key, aes_iv, ecount_buf, &num);
   printf("  got ");
-  hexdump(output, out_len);
+  hexdump(output, sizeof(kPlaintext));
 
 
   /* AES-CTR Decryption */
   OPENSSL_memset(aes_iv, 0, sizeof(aes_iv));
   printf("About to AES-CTR Decrypt ");
   hexdump(output, out_len);
```

Comment:
being pedantic, should be?
```suggestion
  hexdump(output, sizeof(kPlaintext));
```